game-core/src/main/java/games/strategy/triplea/ai/AiPoliticalUtils.java

```diff
@@ -84,13 +84,15 @@ private static boolean wantToPerFormActionTowardsWar(final PoliticalActionAttach
   private static boolean goesTowardsWar(final PoliticalActionAttachment nextAction, final PlayerID p0,
       final GameData data) {
     for (final String relationshipChangeString : nextAction.getRelationshipChange()) {
```

Comment:
Thoughts on creating another `getRelationshipChange` method that returns `List<RelationshipChange>` instead of `List<String>`? Just thinking it reduces the back and forth of getting a list of strings the parsing each one.

Reply:
Funny, I considered doing that when I was reviewing these changes.  I backed off because of the type relationships around the methods that define a `MutableProperty`.  If we did something like this, it would probably have to be a separate method instead of simply changing the return type to `List<RelationshipChange>`.  That then begs the question of what to name the new method. :smile:

I'll investigate further after I prototype changing `RelationshipChange` to hold `PlayerID`s and `RelationshipType` instead of `String`s.